src/style.rs

```diff
@@ -13,6 +14,7 @@ use unicode_segmentation::UnicodeSegmentation;
 #[derive(Clone, Debug)]
 pub struct ProgressStyle {
     tick_strings: Vec<Box<str>>,
+    tick: Cell<u8>,
```

Comment:
Could use some comments. What is `tick`? Why does it need a `Cell` wrapped around it?